validator/src/admin_rpc_service.rs

```diff
@@ -241,6 +271,177 @@ impl AdminRpc for AdminRpcImpl {
         Ok(())
     }
 
+    fn reload_plugin(
+        &self,
+        meta: Self::Metadata,
+        name: String,
+        libpath: String,
+        config_file: String,
+    ) -> BoxFuture<Result<()>> {
+        Box::pin(async move {
+            // Construct channel for plugin to respond to this particular rpc request instance
+            let (tx, rx) = oneshot_channel();
```

Comment:
```suggestion
            let (plugin_result_sender, plugin_result_receiver) = oneshot_channel();
```
`tx` usually means transaction in our code base.
Generally, please spell things out.

Reply:
Makes sense. Will do

Reply:
marked as resolved, as i've changed all tx, rx to sender and receiver